src/AppBundle/Entity/Donation.php

```diff
@@ -77,8 +77,14 @@ class Donation implements GeoPointInterface
      */
     private $createdAt;
 
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $frequency;
```

Comment:
https://github.com/EnMarche/en-marche.fr/pull/1117#discussion_r116957818

I also meant the property, sorry :sweat_smile: